activejob/lib/active_job/enqueuing.rb

```diff
@@ -34,7 +34,54 @@ def perform_later(...)
         enqueue_result
       end
 
+      # Push many jobs onto the queue at once, without running enqueue callbacks.
+      # The parameter +multi_args+ is an array, where each item in the array contains
+      # the arguments for a single job. For example, the following arguments would
+      # enqueue two jobs with three arguments each.
+      #
+      # +multi_args+: [[1, "String", <#SomeModel ...>], [2, "String", <#SomeModel ...>]]
+      #   First job enqueued with [1, "String", <#SomeModel ...>]
+      #   Second job enqueued with [2, "String", <#SomeModel ...>]
+      #
+      # For a job with keyword arguments, the arguments can be passed as a hash:
+      #
+      # +multi_args+: [{ foo: "bar", number: 1 }, { foo: "baz", number: 2 }]
+      #
+      # And for a job with a single argument, no nesting is necessary:
+      #
+      # +multi_args+: ["bar", "baz"]
+      #
```

Comment:
This last calling convention might be confusing if you have a job taking a single positional hash. It would be then interpreted as kwargs.

Also it's not clear how you can call a job that takes both positional and kwargs at the same time.